scipy/stats/_qmc.py

```diff
@@ -1035,6 +1035,23 @@ class LatinHypercube(QMCEngine):
         Dimension of the parameter space.
     centered : bool, optional
         Center the point within the multi-dimensional grid. Default is False.
+
+        .. deprecated:: 1.10.0
+            `centered` is deprecated as of SciPy 1.10.0 and will be removed in
+            1.11.0. Use `scramble` instead. ``centered=True`` corresponds to
```

Comment:
Is there any reason for breaking the convention of two releases?

Reply:
My impatience? 😅 But yes you are correct, thanks for spotting that.
```suggestion
            1.12.0. Use `scramble` instead. ``centered=True`` corresponds to
```